src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets

```diff
@@ -28,30 +28,44 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <!--
-      Configure MVC specific configuration if the application uses a configuration that supports MVC.
+      Resolve MVC specific configuration depending on the presence of MVC specific Razor files.
   -->
-  <PropertyGroup Condition="'$(AddRazorSupportForMvc)' == 'true'">
-    <!--
-      MVC uses a ProvideApplicationPartFactoryAttribute on the generated assembly to load compiled views from assembly. Set this to false, to prevent generating this attribute.
-    -->
-    <GenerateProvideApplicationPartFactoryAttribute Condition="'$(GenerateProvideApplicationPartFactoryAttribute)'==''">true</GenerateProvideApplicationPartFactoryAttribute>
+  <Target Name="_ResolveMvcSpecificRazorConfiguration"
```

Comment:
Should this be something like `_ResolveMvcAssemblyAttributes`? This isn't related to `RazorConfiguration`.